options/options.go

```diff
@@ -229,13 +229,6 @@ func (o *Options) Probes() []string {
 	return o.ProbesToRun
 }
 
-// isExperimentalEnabled returns true if experimental features were enabled via
```

Comment:
nit: Why remove? We'll probably need this flag in the future.

Reply:
`unused` linter